inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/exporter/PrometheusExporterService.java

```diff
@@ -0,0 +1,59 @@
+package rocks.inspectit.oce.core.exporter;
+
+import io.opencensus.exporter.stats.prometheus.PrometheusStatsCollector;
+import io.opencensus.exporter.stats.prometheus.PrometheusStatsConfiguration;
+import io.prometheus.client.exporter.HTTPServer;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.springframework.stereotype.Component;
+import rocks.inspectit.oce.core.config.model.InspectitConfig;
+import rocks.inspectit.oce.core.service.DynamicallyActivatableService;
+
+import static io.prometheus.client.CollectorRegistry.defaultRegistry;
+
+/**
+ * Service for the Prometheus OpenCensus exporter.
+ * Can be dynamically started and stopped using the exporters.metrics.prometheus.enabled configuration.
+ */
+@Component
+@Slf4j
+public class PrometheusExporterService extends DynamicallyActivatableService {
+
+    private HTTPServer prometheusClient = null;
+
+    public PrometheusExporterService() {
+        super("exporters.metrics.prometheus");
```

Comment:
wouldn't it be better to pass to complete config name to the superclass `exporters.metrics.prometheus.enabled`? Then you are not obligated to have property ending in `...enabled`.. Or how does this work actually?